### ISSUE TITLE
Can/shall we have a page of contributors?

### ISSUE THREAD

**Issue by niruvt** [learnlatex/learnlatex.github.io#189]
Right now I can see https://www.learnlatex.org/AUTHORS page active without language independently, but interestingly I haven't found the link for this page on any of the webpages of our site. I think we can have a detailed AUTHORS page rather than just listing down the contributors.

(e.g. 
1. Core development: 
   1. Person A
   1. Person B
   1. Person C
1. Translations
   1. Vietnamese
      1. Person D
   1. Marathi
      1. Person E

)

This can be helpful for citing our work as there will be an authentic documentary evidence of the contribution. Of course there is one on the git repository, I am aware of it, but not everyone knows how to explore the source-code, hence having it on the website might be a good idea.

**Comment by Skillmon** [comment]
Imho, attributing different sections to the individual authors isn't worth the hassle. I'm fine with citations only saying "by Learn LaTeX", "by the Learn LaTeX team" or anything like that. But if others think this is a good idea it's fine for me.

**Comment by niruvt** [comment]
@Skillmon, I agree, but often to be particular few people like to be particular like who is the author, who is the translator and things like that. Also this is not just about that; sometimes it is quite necessary to cite the particulars of the work. e.g. Authored by The LearnLaTeX team, translated by Person X. Also there should be proper attribution. e.g. In the contributors' field my name is there. Attributing the site's work to the team reflects that I am also one of the "author", whereas actually I am not! This has two problems. 1) It gives me credit for a work that I actually have not done. 2) I am held responsible of the content. Both of this is not ideal IMO. Hence I still think that the contributions should be properly documented.

**Comment by davidcarlisle** [comment]
While it could perhaps be more prominent there is a link to the autor list page on the github-pages view.  

every page has a link [Contacts](https://www.learnlatex.org/en/contact) whioch links to [Contribiting](https://www.learnlatex.org/CONTRIBUTING)  which in turn links to the [contributors list](https://www.learnlatex.org/AUTHORS)

**Comment by niruvt** [comment]
@davidcarlisle

Thanks, I didn't know about this. Though I think my point regarding making the page more specific and detailed is not yet answered. Also having a link to the authors page at least on the contacts page would be much better as maybe no one would go to the contributing-guidelines page to find the list of contributors. It doesn't seem obvious to me at least.

**Comment by davidcarlisle** [comment]
In the above commit I added a possible table to address this in an author-role branch so the author list has a role as well as a simple list of authors. I'm not totally convinced it's a good idea: adding roles without writing a complete essay about who did what has potential to offend people by omission. But I'd be interested in comments.

**Comment by jejust** [comment]
I feel that's even more misleading to sum up contributions in 2 to 7 words than not to say anything... From the mere list, it's clear that contributors are listed in alphabetical order, and that contributions are not equal. As soon as you provide details, it gives a wrong impression of precision and that hides the fact that a few persons are leading the project from the beginning, with a strong involvement, while others just gave a hand by translating, or just suggested small changes.

On the other hand, the table can help people find the right person to contact when an error is discovered in a translation.

**Comment by davidcarlisle** [comment]
@jejust  yes that was my worry as well, it's difficult to balance not writing an essay for each person, and not offending people by not mentioning what they did.  I'm in no hurry to merge that, I'd be interested in comments from other people. 

On the other hand, while making the table version I spotted a couple of people not mentioned at all in the current list on the main branch so I'll probably update that list.

**Comment by Skillmon** [comment]
> On the other hand, the table can help people find the right person to contact when an error is discovered in a translation.

While I'm grateful for all the translators' work, I don't think they should be the direct contact person for translation errors. Instead I think those should just be handled as any other error: Open a ticket on Github, we have a `translation` label for issues. The people who translated to a language in the first place might lose interest, don't have enough free time, or what else.

**Comment by niruvt** [comment]
> As soon as you provide details, it gives a wrong impression of precision and that hides the fact that a few persons are leading the project from the beginning, with a strong involvement, while others just gave a hand by translating, or just suggested small changes.

I also found the roles mentioned in the list a bit too specific and thus leading to an unclear picture of the weightage of the work. May be we can just separate the key-roles. e.g. 

* The design of the coursework:
  (A list of all the contributors who have made some contribution to the _content_ of the course. May be some names from the translation section can repeat here.)
* Website development and implementation:
  Jonas Jaceck
* Translations
  - French: Jérémy Just
  - Marathi: निरंजन

The problem that Jérémy rightly points out with the "extra" precision might be resolved with this kinda list and the team will not suffer from the aggregation of the roles.
